docs/design/clustering.md

```diff
@@ -0,0 +1,56 @@
+# Clustering in Kubernetes
+
+
+## Overview
+The term "clustering" refers to the process of having all members of the kubernetes cluster find and trust each other.  There are multiple different ways to achieve clustering with different security and usability profiles.  This document attempts to lay out the user experiences for clustering that Kubernetes aims to address.
+
+Once a cluster is established, the following is true:
+
+1. **Master -> Node**  The master needs to know which nodes can take work and what their current status is wrt capacity.
+  1. **Location** The master knows the name and location of all of the nodes in the cluster.
```

Comment:
Not sure what you are meaning by pathway.  Adding some clarification here that we need consistency so that we can verify certificates, at the least.

Suggest some language and I'll include it :)